pkg/conductor_client/conductor_http_client/api_client.go

```diff
@@ -216,10 +216,10 @@ func (c *APIClient) mustRefreshToken() bool {
 }
 
 func (c *APIClient) refreshToken() {
-	log.Debug("Refreshing authentication token")
+	logrus.Debug("Refreshing authentication token")
 	token, response, err := c.getToken()
 	if err != nil {
-		log.Warn(
+		logrus.Warning(
```

Comment:
why are we changing from log to logrus?

Reply:
Applied new standard, import `logrus` as `log` alias for entire repo